tests/components/v2c/test_config_flow.py

```diff
@@ -0,0 +1,61 @@
+"""Test the V2C config flow."""
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.v2c.config_flow import CannotConnect
+from homeassistant.components.v2c.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+pytestmark = pytest.mark.usefixtures("mock_setup_entry")
```

Comment:
Can be removed iirc

Reply:
Required for ```assert len(mock_setup_entry.mock_calls) == 1```

Reply:
You're already passing `mock_setup_entry` as parameter in the test